src/web/core/src/main/java/org/geoserver/web/wicket/CodeMirrorEditor.js

```diff
@@ -3,7 +3,8 @@ var editor = CodeMirror.fromTextArea(textarea, {
     mode: '$mode',
     theme: 'default',
     lineWrapping: true,
-    lineNumbers: true
+    lineNumbers: true,
+    extraKeys: {"Ctrl-Space": "autocomplete"}
```

Comment:
I am wondering if codemirror might be playing some magic already... like, all the codemirror site demos hard-code CTRL-Space, like this one:
https://codemirror.net/demo/html5complete.html
Do they not work on OSX?